### PR TITLE
Added scenario to check activity from public user

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Add bug demonstration scenario for #1032

Part of #1019 See https://github.com/owncloud/activity/issues/1019#issuecomment-966086091

### CODE REVIEW

File: tests/acceptance/features/bootstrap/ActivityContext.php [hunk 1]
```diff
@@ -70,12 +72,53 @@ public function activityWithIndexShouldMatch(
...
+		string $index,
+		string $user,
+		TableNode $expectedProperties
+	): void {
+		$responseDecoded = $this->getActivitiesForUser($user);
+		$this->assertActivityIndexContent($user, $index, $responseDecoded, $expectedProperties);
+	}
+
+	/**
+	 * @Given /^as user "([^"]*)" the activity number (\d+) for "([^"]*)" should match these properties:$/
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/ActivityContext.php:95]:
```suggestion
	 * @Then /^as user "([^"]*)" the activity number (\d+) for "([^"]*)" should match these properties:$/
```

This looks like a Then step.

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiActivity-master-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/2314/25/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiActivity-master-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/2319/3/1

**Comment by kiranparajuli589** [comment]
blocked because activity is not listed for rename using `/core/remote.php/webdav/parent/textfile.txt HTTP/1.1`

File: tests/acceptance/features/bootstrap/ActivityContext.php (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/ActivityContext.php]: fixed

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiActivity-master-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/2326/25/1

**Comment by individual-it** [comment]
@phil-davis @kiranparajuli589 should we have bug-demo tests for this?

**Comment by kiranparajuli589** [comment]
> @phil-davis @kiranparajuli589 should we have bug-demo tests for this?

@individual-it I've refactored the code to include a bug demonstration scenario.

**Comment by individual-it** [comment]
rebased on master

**Review by phil-davis — APPROVED** [review]

**Pull request merged**